runtime/tl/rpc_request.h

```diff
@@ -47,6 +47,7 @@ class RpcRequestResult : public ManagedThroughDlAllocator {
 
   virtual class_instance<C$VK$TL$RpcResponse> fetch_typed_response() = 0;
   virtual std::unique_ptr<tl_func_base> extract_untyped_fetcher() = 0;
+  virtual bool is_typed() const = 0;
```

Comment:
nit: you can use public constant bool filed and set it from constructor, this way the virtual function call can be avoided

Reply:
fixed